src/test/resources/application.properties

```diff
@@ -14,5 +14,8 @@ service.tm.username=
 service.tm.password=
 service.mocktm.url=http://localhost:8000
 service.rabbit.url=localhost
```

Comment:
Should the csvfile be part of the test?

Reply:
As discussed, acceptance test for CCS has now been completed.